src/plugins/kibana/public/management/index.js

```diff
@@ -42,6 +42,16 @@ uiModules
       if ($scope.section) {
         $scope.section.items.forEach(item => {
           item.class = `#${$location.path()}`.indexOf(item.url) > -1 ? 'active' : undefined;
+          // Disable link if path is not set
+          if (!item.path) {
+            delete item.url;
+          }
+          item.items.forEach(childItem => {
+            // Disable link if path is not set
+            if (!childItem.path) {
+              delete childItem.url;
```

Comment:
I like the idea of not persisting `path` and using it purely as a convenience option to generate URL. I'm guessing then this code in the ManagementSection class constructor:

    if (!options.url) {
      options.url = '#/management/' + options.path || '';
    }

would change to:

    if (!options.url && options.path) {
      options.url = '#/management/' + options.path;
    }

(Otherwise, with the current code we end with `url` set to '#/management/undefined' if neither `url` nor `path` are set by the registerer).